config.toml

```diff
@@ -67,58 +67,59 @@ summaryLength = 70
     parent     = "section.ops"
 
 [[menu.main]]
-    name       = "Grafana"
+    name       = "Metrics"
     url        = "https://metric.sonix.network"
-    weight     = 1
+    weight     = 2
     parent     = "section.services"
 
 [[menu.main]]
     name       = "PeeringDB"
     url        = "https://peeringdb.com/ix/3607"
-    weight     = 2
+    weight     = 3
     parent     = "section.services"
 
 [[menu.main]]
     name       = "IXP Manager"
     url        = "https://manager.sonix.network/"
-    weight     = 3
+    weight     = 4
     parent     = "section.services"
 
 [[menu.main]]
-    name       = "MEMBERS"
+    name       = "SONIX Friends"
     identifier = "menu.members"
     url        = "/members/"
-    weight     = 5
+    weight     = 60
 
 [[menu.main]]
     name       = "FAQ"
     identifier = "menu.faq"
     url        = "/faq/"
-    weight     = 6
+    weight     = 70
 
 [[menu.main]]
     name       = "Status"
     identifier = "menu.status"
     url        = "https://status.sonix.network"
-    weight     = 7
+    weight     = 1
```

Comment:
Status page links are usually found at the right, how about we move it? and/or make it easier to find?

Reply:
I moved status page under IX/Services and added it to the top bar. Isn't that enough?

Reply:
That made is less visible? WDYM?
I think moving it so it's on the right of the top bar could help

![image](https://github.com/user-attachments/assets/69e27e1d-d71e-4498-9d0c-eea11cbeb9a0)

![image](https://github.com/user-attachments/assets/62341958-cf4a-4812-b72b-168d68ade308)

Reply:
It's fixed and I agree. I thought you were referring to the main menu due to your excerpt from config.toml. I created new entrys in config.toml for the top bar arrangement:

```[params.topbar.services]
    links = [
        { name = "Metrics", url = "https://metric.sonix.network" },
        { name = "PeeringDB", url = "https://peeringdb.com/ix/3607" },
        { name = "IXP Manager", url = "https://manager.sonix.network/" },
        { name = "Status", url = "https://status.sonix.network" }
    ]
    ```